Library/TeamTalkLib/teamtalk/PacketLayout.inl

```diff
@@ -68,7 +68,6 @@ CryptPacket< PACKETTYPE, PACKET_KIND_CRYPT, PACKET_KIND_DECRYPTED >::CryptPacket
     int encrypt_len = 0, tmpLen = 0;
     EVP_CIPHER_CTX* aesEncCtx = EVP_CIPHER_CTX_new();
     cipher_guard g(aesEncCtx);
-    EVP_CIPHER_CTX_init(aesEncCtx);
```

Comment:
How come you remove this call?

The OpenSSL documentation states:

_The EVP cipher routines are a high level interface to certain symmetric ciphers.

EVP_CIPHER_CTX_init() initializes cipher contex ctx.

EVP_EncryptInit_ex() sets up cipher context ctx for encryption with cipher type from ENGINE impl. ctx must be initialized before calling this function._